docs/reference/datatiers.asciidoc

```diff
@@ -88,20 +88,15 @@ For resiliency, indices in the cold tier can rely on
 Once data is no longer being queried, or being queried rarely, it may move from
 the cold tier to the frozen tier where it stays for the rest of its life.
 
-The frozen tier uses <<searchable-snapshots,{search-snaps}>> to store and load
-data from a snapshot repository. Instead of using a full local copy of your
-data, these {search-snaps} use smaller <<shared-cache,local caches>> containing
-only recently searched data. If a search requires data that is not in a cache,
-{es} fetches the data as needed from the snapshot repository. This decouples
-compute and storage, letting you run searches over very large data sets with
-minimal compute resources, which significantly reduces your storage and
-operating costs.
-
-The <<ilm-index-lifecycle, frozen phase>> automatically converts data
-transitioning into the frozen tier into a shared-cache searchable snapshot.
-
-Search is typically slower on the frozen tier than the cold tier, because {es}
-must sometimes fetch data from the snapshot repository.
+The frozen tier uses <<searchable-snapshots,partially mounted {search-snaps}>>
+to store and load data from a snapshot repository. When an index transitions into
+the <<ilm-index-lifecycle,frozen phase>>, {ilm-init} automatically converts it
+to a partially mounted {search-snap} on the frozen tier.
+
+Partially mounted {search-snaps} let you search over large data sets with fewer
+compute resources, reducing both your storage and operating costs. Because {es}
+must sometimes fetch data from the snapshot repository, searches on the frozen
+tier are typically slower than on the cold tier.
```

Comment:
I refactored this section to deduplicate content already covered in the mount options section of the searchable snapshots page.